docs/sphinx/developers/java-library.txt

```diff
@@ -1,7 +1,43 @@
 Using Bio-Formats as a Java library
 ===================================
 
-If you wish to make use of Bio-Formats within your own software, you can
+Bio-Formats as a Maven dependency
+-----------------
```

Comment:
These should have the same size as the title otherwise Sphinx will fail with the warnings treated as errors options turned on.